bot.rb

```diff
@@ -14,12 +14,19 @@ class Bot < Summer::Connection
 
   BALL8_TIPS = YAML.load(File.read(File.join(File.dirname(__FILE__), 'config', '8ball.yml')))
   REASON_TIPS = YAML.load(File.read(File.join(File.dirname(__FILE__), 'config', 'reasons.yml')))
+  OPS = YAML.load(File.read(File.join(File.dirname(__FILE__), "config", "ops.yml")))
   CONFIG = YAML::load_file("config/summer.yml")
   def did_start_up
     ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'] || config['database'])
     privmsg("Bot started up at #{Time.now.strftime("%d %B %Y %H:%M")}", CONFIG['owner'])
     @pastebin_dumbass = {}
   end
+  
+  def ops_command(sender, channel, message, opts={})
+    OPS[channel].each do |operator|
+      privmsg("In #{channel}: #{sender} -> #{message}", operator)
```

Comment:
I would prefer a channel message rather than a PM. Like:

"Radar, sevenseacat: your presence is requested by <nick>"